script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_10.txt

```diff
@@ -3,44 +3,44 @@
 <Page2>
 Without anyone waking me up,
 Without hearing anything,
-Vaguely without any reason,# I wonder when was the last time I woke up in the middle of my sleep.
+Seemingly without any reason,# I wonder when it last was that I woke up.
```

Comment:
Uhh this loses quite a bit of the original, and doesn't really make sense anymore
> 眠りの途中で目が覚めたのはいつ以来だろう。